student-source/students/accessing-codio/paying.rst

```diff
@@ -62,13 +62,37 @@ If you have any problems contact us through your Codio account or email us at he
 
-We have heard of a few cases, especially from US banks where they do not allow international transactions by default but when you are able to explain to them what you are looking to do, they usually are able to allow it and you can then subscribe with that card but if they won't, try a different card.
+It should be understood that Codio has no control over such transaction refusal. Your credit card company would even not disclose to Codio the exact reason for your transaction to fail.
 
-If you are seeing General error in Cardinal, this error can be caused due to adblockers or cookies being blocked
+This means that unfortunately Codio cannot assist in solving the issue which is strictly a matter between the card issuing credit card company (who refused to authorize the transaction) and the credit card holder (who initiated the process to have the amount of the transaction charged to his/her credit card).
+
+In case of a transaction failure, Codio recommends to first try using an alternative credit card, or to give it another try with the same credit card again but at a later time. In most cases this would solve the issue.
+
+Do carefully check the card details before confirming the transaction for processing. Also ensure your card is actually a credit card and not just a debit card. 
+
+Should the transaction continue to fail you might want to get in touch with your credit card company. Your credit card company will be able to check the transaction log for your credit card and help fix the problem. In some cases your credit card company would need to pre-authorize the transaction for you before you can process the transaction again.
+
+A list of common reasons why payments fail:
+
+- Your credit card limit was reached (insufficient funds)
+- Check your credit limit. You might have reached your daily or total credit card limit on the day you tried to charge your card. If this is the case, ask your credit card company to increase your limit.
+- The charge exceeded the maximum amount allowed for a single charge
+- Check the maximum amount your card can be charged at a single time. If the declined charge is higher than this amount, ask your credit card company to increase the limit
+- Your card reached the maximum number of charges allowed in a period: Check how many times your card can be charged in given time period. If you reached this amount, ask your bank to increase the limit, wait until your card can be charged again, or make a payment with a different payment method.
+- Your card doesn't accept charges from an online source: Make sure your card allows online transactions. If it doesn't, talk to your credit card company about allowing these charges. Or use another card
```

Comment:
Your card doesn't accept charges from an online source: Make sure your card allows online transactions. If it doesn't, talk to your credit card company about allowing these charges or use another card.